workshop.wwwapi/Repository/IRepository.cs

```diff
@@ -5,9 +5,16 @@ namespace workshop.wwwapi.Repository
     public interface IRepository
     {
         Task<IEnumerable<Patient>> GetPatients();
+        Task<PatientDto> GetPatientById(int id);
+        Task<Patient> CreatePatient(PatientDto patientDto);
```

Comment:
don't pass the DTOs into the CreatePatient, instead pass the raw data like the patient name, age, etc as individual variables

Reply:
we don't want the repo layer to use the endpoint layer objects (dtos)